nh_eobs_demo_mental_health/tests/nh_clinical_wardboard/test_get_last_finished_pme_sql_view.py

```diff
@@ -0,0 +1,100 @@
+from openerp.tests.common import TransactionCase
+
+
+class TestGetLastFinishedPMESQLView(TransactionCase):
+    """
+    Test that the get_last_pme SQL view is working correctly
+    """
+
+    def get_pmes_for_state(self, state):
+        """
+        Get Patient Monitoring Exceptions activities in a state
+
+        :param state: State of activity (completed, cancelled, started)
+        :return: recordset of nh.activity
+        """
+        activity_model = self.env['nh.activity']
+        return activity_model.search([
+            ['data_model', '=', 'nh.clinical.patient_monitoring_exception'],
+            ['state', '=', state]
+        ])
+
+    def get_last_finished_pmes(self, spell):
+        """
+        Use the last_finished_pme SQL view to get the last PMEs in a state
+        of completed or cancelled
+        :param spell: instance of nh.clinical.spell from recordset
+        :return: dictionary of values from cursor
```

Comment:
Bit confused by "last finished pme**s**" (plural). Can't there only be one last pme?